static/app/views/performance/transactionSummary/transactionEvents/index.tsx

```diff
@@ -0,0 +1,181 @@
+      // If there is no transaction name, redirect to the Performance landing page
+      browserHistory.replace({
+        pathname: `/organizations/${organization.slug}/performance/`,
+        query: {
+          ...location.query,
+        },
+      });
+      return null;
+    }
+
+    const shouldForceProject = eventView.project.length === 1;
+    const forceProject = shouldForceProject
+      ? projects.find(p => parseInt(p.id, 10) === eventView.project[0])
+      : undefined;
+    const projectSlugs = eventView.project
+      .map(projectId => projects.find(p => parseInt(p.id, 10) === projectId))
+      .filter((p: Project | undefined): p is Project => p !== undefined)
+      .map(p => p.slug);
+    return (
+      <SentryDocumentTitle
+        title={this.getDocumentTitle()}
+        orgSlug={organization.slug}
+        projectSlug={forceProject?.slug}
+      >
+        <Feature
+          features={['performance-events-page']}
+          organization={organization}
+          renderDisabled={this.renderNoAccess}
+        >
+          <GlobalSelectionHeader
+            lockedMessageSubject={t('transaction')}
+            shouldForceProject={shouldForceProject}
+            forceProject={forceProject}
+            specificProjectSlugs={projectSlugs}
+            disableMultipleProjectSelection
+            showProjectSettingsLink
+          >
+            <LightWeightNoProjectMessage organization={organization}>
+              <EventsPageContent
+                location={location}
+                eventView={eventView}
+                transactionName={transactionName}
+                organization={organization}
+                isLoading={false}
+                totalValues={null}
+                projects={projects}
+                spanOperationBreakdownFilter={this.state.spanOperationBreakdownFilter}
+              />
+            </LightWeightNoProjectMessage>
+          </GlobalSelectionHeader>
+        </Feature>
+      </SentryDocumentTitle>
+    );
+  }
+}
+
+function generateEventsEventView(
+  location: Location,
+  transactionName: string | undefined
+): EventView | undefined {
+  if (transactionName === undefined) {
+    return undefined;
+  }
+  // Use the user supplied query but overwrite any transaction or event type
+  // conditions they applied.
+  const query = decodeScalar(location.query.query, '');
+  const conditions = tokenizeSearch(query);
+  conditions
+    .setTagValues('event.type', ['transaction'])
+    .setTagValues('transaction', [transactionName]);
+
+  Object.keys(conditions.tagValues).forEach(field => {
+    if (isAggregateField(field)) conditions.removeTag(field);
+  });
+
+  // Default fields for relative span view
+  const fields = [
+    'id',
+    'user.display',
+    SPAN_OP_RELATIVE_BREAKDOWN_FIELD,
+    'transaction.duration',
+    'trace',
+    'timestamp',
+    'spans.total.time',
+  ];
+  fields.push(...SPAN_OP_BREAKDOWN_FIELDS);
+
+  return EventView.fromNewQueryWithLocation(
```

Comment:
Should this `EventView` impose a sort order? Otherwise, the results are just going to be a seemingly random set of events.

Reply:
Hey Tony, I've added a simple `orderby: '-timestamp'` to the query. This should work for now. Once I get started on the header sort and resize ticket, I should have a more dynamic solution.

Reply:
That works for me 👍